server/app/playbook_run_service.go

```diff
@@ -2612,6 +2612,21 @@ func (s *PlaybookRunServiceImpl) PublishRetrospective(playbookRunID, publisherID
 	return nil
 }
 
+func (s *PlaybookRunServiceImpl) ToggleRetrospective(playbookRunID string, retrospective RetrospectiveUpdate) error {
+	playbookRunToPublish, err := s.store.GetPlaybookRun(playbookRunID)
+	if err != nil {
+		return errors.Wrap(err, "failed to retrieve playbook run")
+	}
+
+	// Update the text to keep syncronized
+	playbookRunToPublish.RetrospectiveEnabled = !playbookRunToPublish.RetrospectiveEnabled
+	if err = s.store.UpdatePlaybookRun(playbookRunToPublish); err != nil {
+		return errors.Wrap(err, "failed to update playbook run")
+	}
+
```

Comment:
Updated run is not pushed back to WebSocket.
```suggestion
	s.poster.PublishWebsocketEventToChannel(playbookRunUpdatedWSEvent, playbookRunToPublish, playbookRunToPublish.ChannelID)

```